game/scripts/npc/abilities/undying_decay.txt

```diff
@@ -26,7 +26,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "10 8 6 4 4 4"
+    "AbilityCooldown"                                     "8 7 6 5 4.5 4" //OAA
```

Comment:
I was thinking that str steal needs a rescale, not this.

Reply:
Yeah something is wrong with decay, I wasn't sure what it was. Str steal is integer values so it seemed weird to change it